custom_components/tuya_local/devices/ustellar_ut99911_rgb.yaml

```diff
@@ -0,0 +1,174 @@
+name: Smart Monitor Light Bar
+products:
+  - id: oqtk22eg6jxorl78
+    name: Ustellar UT99911-RGB
+primary_entity:
+  entity: light
+  name: Front LED (White)
+  icon: "mdi:wall-sconce-flat-outline"
+  dps:
+    - id: 101
+      type: boolean
+      name: switch
+    - id: 22
+      name: brightness
+      type: integer
+      range:
+        min: 10
+        max: 1000
+      mapping:
+        - scale: 3.92
+    - id: 23
+      type: integer
+      name: color_temp
+      mapping:
+        - target_range:
+            min: 2700
+            max: 6500
+      range:
+        min: 0
+        max: 1000
+secondary_entities:
+  - entity: light
+    name: Rear LED (RGB)
+    icon: "mdi:wall-sconce-flat-variant-outline"
+    dps:
+      - id: 20
+        type: boolean
+        name: switch
+      - id: 21
+        type: string
+        name: color_mode
+        mapping:
+          - dps_val: dynamic_mod
+            value: Dynamic
+          - dps_val: color
+            value: hs
+          - dps_val: scene_mod
+            value: Scene
+          - dps_val: music
+            value: Music
+      - id: 24
+        name: rgbhsv
+        type: hex
+        format:
+          - name: h
+            bytes: 2
+            range:
+              min: 0
+              max: 360
+          - name: s
+            bytes: 2
+            range:
+              min: 0
+              max: 1000
+          - name: v
+            bytes: 2
+            range:
+              min: 0
+              max: 1000
+      - id: 103
+        name: line_sequence_adjustment
+        type: integer
+      - id: 104
```

Comment:
This can be removed, as it is listed below as a separate entity, and is not part of the light definition. (line_sequence_adjustment on the other hand does not appear elsewhere, so if it doesn't make sense as a separate entity can stay as an extra attribute here for reference).

Reply:
Removed 104 - left 103 in there. This was based on the blitzwolf config but no need for that attribute to exist.